tests/test_enum.py

```diff
@@ -0,0 +1,12 @@
+from flask_common.enum import Enum
+
+class TestEnum(Enum):
+    A = 'a'
+    B = 'b'
+
+
+def test_enum():
```

Comment:
Add a docstring/comment explaining why we're doing the seemingly same check twice.